tests/detect-bypass/test.yaml

```diff
@@ -0,0 +1,22 @@
+args:
+- --set stream.midstream=true
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 1
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 2
+- filter:
+    count: 2
+    match:
+      event_type: flow
```

Comment:
With the change in direction for packet 2 (mentioned in another comment), this should also change, as this is supposed to be a single flow.

Reply:
indeed